ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -426,18 +507,18 @@ def test_random_tucker(
             assert np.prod(shape) == np.prod(x_gt.shape)
 
     else:
-        core = helpers.flatten_and_to_np(ret=ret_np[0], backend=backend_fw)
+        weights = helpers.flatten_and_to_np(ret=ret_np[0], backend=backend_fw)
         factors = helpers.flatten_and_to_np(ret=ret_np[1], backend=backend_fw)
-        core_gt = helpers.flatten_and_to_np(
+        weights_gt = helpers.flatten_and_to_np(
             ret=ret_from_gt_np[0], backend=test_flags.ground_truth_backend
         )
         factors_gt = helpers.flatten_and_to_np(
             ret=ret_from_gt_np[1], backend=test_flags.ground_truth_backend
         )
 
-        for c, c_gt in zip(core, core_gt):
-            assert np.prod(c.shape) == np.prod(rank)
-            assert np.prod(c_gt.shape) == np.prod(rank)
+        for w, w_gt in zip(weights, weights_gt):
+            assert len(w) == rank
+            assert len(w_gt) == rank
```

Comment:
Why are we changing this? `TuckerTensor` consists of a core tensor and a list of factor matrices.  I dont't think there was anything wrong with this, but maybe I'm missing something, could you please explain why you made this change?

Reply:
this was failing because it was missing a `test_gradients=st.just(False),` but when I did those changes the tests passed.
now I added the missing attribute and revert the changes and now everything is passing

Reply:
Sure, thanks for adding it :pray: